build/test-util/CMakeLists.txt

```diff
@@ -57,9 +57,19 @@ FetchContent_Declare(
   GIT_TAG ${TRITON_CORE_REPO_TAG}
   GIT_SHALLOW ON
 )
+FetchContent_Declare(
```

Comment:
We do this inside the test/example backends. I think we can remove this change and keep the one in CMakeLists.txt